src/native/hash/bitfn.h

```diff
@@ -22,8 +22,7 @@
  * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#ifndef BITFN_H
-#define BITFN_H
+#pragma once
```

Comment:
Do we really need so much OT comments and github reactions to try to make programmers write C standard compliant code ? This place is turning into the worst kind of pointless forum.